gramhopper/triggers/filter_triggers.py

```diff
@@ -33,32 +34,26 @@ def __init__(self, lang):
         super().__init__(Filters.language(lang))
 
 
+class _MessageTypeFilterBasedTrigger(FilterBasedTrigger):
+    def __init__(self, message_type=None):
+        try:
+            subfilters = message_type.split('.')
+            message_type_filter = reduce(
```

Comment:
Took me a while to get how it works, but that's really nice!
I think I've done something alike (not as nice as your code), can they use the same code?

Reply:
They can't since `Filters` is not a DictEnum.